src/client/datascience/notebook/creation/notebookCreator.ts

```diff
@@ -48,7 +45,6 @@ export class NotebookCreator {
             label: JVSC_EXTENSION_DisplayName
         });
         const placeHolder = DataScience.placeHolderToSelectOptionForNotebookCreation();
-        traceInfoIf(IS_CI_SERVER, `Display quick pick for creation of notebooks ${items.length}`);
```

Comment:
Sorry @IanMatthewHuff I thought i removed this in the previous PR, removing here.

Reply:
NBD I filed a code health bug to just clean this usage up all up. No need to sweat it now.